src/developer-quickstart.md

```diff
@@ -0,0 +1,441 @@
+
+We'll be deploying to the Fuel testnet. In order to deploy a contract, you need to have a wallet to sign the transaction. You can optionally pass the flag `--unsigned` to deploy a contract without a signature.
+
+By default fuel-core runs without UTXO validation, which means you can run unsigned transactions. This allows you to send invalid inputs to emulate different conditions.
+
+### Install the Wallet CLI
+
+Follow [these steps to set up a wallet to sign transactions](https://github.com/FuelLabs/forc-wallet#forc-wallet).
+
+After typing in a password, be sure to save the mnemonic phrase that is output.
+
+With this, you'll get a fuel address that looks something like this: `fuel1efz7lf36w9da9jekqzyuzqsfrqrlzwtt3j3clvemm6eru8fe9nvqj5kar8`. Save this address as you'll need it to sign transactions when we deploy the contract.
+
+#### Deploy To Testnet
+
+Now that you have a wallet, you can deploy with `forc deploy` and passing in the testnet endpoint like this:
+
+`forc deploy --url https://node-beta-1.fuel.network/graphql --gas-price 1`
+
+> Note: We set the gas price to 1. Without this flag, the gas price is 0 by default and the transaction will fail.
+
+The terminal will ask for the address of the wallet you want to sign this transaction with, paste in the address you saved earlier, it looks like this: `fuel1efz7lf36w9da9jekqzyuzqsfrqrlzwtt3j3clvemm6eru8fe9nvqj5kar8`
+
+The terminal will output your contract id. Be sure to save this as you will need it to build a frontend with the Typescript SDK.
+
+The terminal will output a `transaction id to sign` and prompt you for a signature. Open a new terminal tab and view your accounts by running `forc wallet list`. If you followed these steps, you'll notice you only have one account, `0`.
+
+Grab the `transaction id` from your other terminal and sign with a specified account by running the following command:
+
+`forc wallet sign` + `[transaction id here, without brackets]` + `[the account number, without brackets]`
+
+Your command should look like this: `forc wallet sign 16d7a8f9d15cfba1bd000d3f99cd4077dfa1fce2a6de83887afc3f739d6c84df 0`
+
+Enter your password when prompted, and you'll get back a `signature`. Save that signature, and return to your other terminal window, and paste that in where its prompting you to `provide a signature for this transaction`.
+
+Finally, you will get back a `TransactionId` to confirm your contract was deployed. With this ID, you can head to the [block explorer](https://fuellabs.github.io/block-explorer-v2/) and view your contract.
+
+## Create a Frontend to Interact with Contract
+
+Now we are going to
+
+1. **Initialize a React project.**
+2. **Install the `fuels` SDK dependencies.**
+3. **Modify the App.**
+4. **Run our project.**
+
+### Initialize a React project
+
+To split better our project let's create a new folder `frontend` and initialize our project inside it.
+
+Inside `counter-dapp` run;
```

Comment:
On this step, we lose the user on the folder as following the steps leads to creating the folder front inside the counter_dapp. I think it is ok to do like this we just need to remove references for other folders. I think this happened because we missed a step to create a folder at the beginning. It should be before line 46.

Reply:
In this step my folder three looked like this;

```
├── Cargo.lock
├── Cargo.toml
├── Forc.lock
├── Forc.toml
├── frontend
|  ├── README.md
|  ├── node_modules
|  ├── package-lock.json
|  ├── package.json
|  ├── public
|  ├── src
|  └── tsconfig.json
├── out
|  └── debug
├── src
|  └── main.sw
├── target
|  ├── CACHEDIR.TAG
|  ├── debug
|  └── tmp
└── tests
   └── harness.rs
```

Reply:
good catch, fixed!